routers/web/repo/repo.go

```diff
@@ -280,6 +281,16 @@ func CreatePost(ctx *context.Context) {
 	handleCreateError(ctx, ctxUser, err, "CreatePost", tplCreate, &form)
 }
 
+func canPin(ctx *context.Context) bool {
```

Comment:
If I see that correctly, this method can also be a part of `services`.

Reply:
Can you point me at an example file where this would go?

Reply:
I guess `services/user/user.go` would be a good place for this function...